Monika-After-Story/MonikaModDev
# I deleted our idle sprites

Implemented a new system for idle sprite management . Instead of ATL, this uses its own class `MASMoniIdleDisp`. 

Why?
- `ATL` is buggy. For example it likes to restart when we don't need, but refuses to restart when we do need it to. It also has issues with other transforms and transitions.
- `ATL` gives no control over it, the new disp allows to change exp with one method.
- With `ATL` we can't really get status of `ATL`, the disp allows us to get current exp, its duration, and much more.
- For `ATL` we kinda had to duplicate code, but now adding a new expression is a matter of writing one line.
- `ATL` would be a pain to use with the mood system. This disp is kinda a step towards moods.
- I think my disp is more optimized than `ATL` + `ConditionSwitch`.

The new disp completely mimics the old ATL regarding how it looks. Although, for ATL sprites "total weight" was always `1`, for the disp total weight depends on the weight of all exps. So this may need to be adjusted, since I used the old values for weights.

New:
- `MASMoniIdleDisp` - the new displayable to handle idle sprite.
- `MASMoniIdleExp` - a class to represent a single expression.
- `MASMoniIdleExpGroup` - a class to represent a group of exps. This is used when we need to show multiple exp one by one. It's a subclass of `MASMoniIdleExp`, so they share some props and methods (and basically can be treated the same).
- `MASMoniIdleExpRngGroup` - a subclass of `MASMoniIdleExpGroup` for rng based img selection.
- New return key: `idle_exp` can be used to force a temp idle exp. The exp can be generated on the fly or being retrieved by a tag. For example: `return "idle_exp: 5eubsa, 5"` or `return "idle_exp: happy"`. If the tag doesn't exist, nothing will happen. If there's multiple exp with the same tag, we use weighted choice to select one. I haven't added any predefined groups yet.

Changes:
- I added some more idle sprites. If we want some more, then it's good time to add them.
- `monika idle` is now predicted after the splash screen.
- We can use `monika idle` for spaceroom dissolving again. Hooray for no `ATL` seizures.

### Testing:
- I had to change the wink transform, I haven't noticed any issues, but I'd ask to pay some attention to it.
- Please, verify that we have good idle sprites. Especially their parameters like weights/conditionals/exp codes.
- Verify loading times are fine, no freezes during switching from topic into idle, etc.
- Verify that `conditional`'d exps are only being shown if the condition is met, same for `aff_range`.
- Give the new system a try. Force a new exp, try to add or remove one during runtime.
- Play with the return keys, make sure we have everything we want.
- Verify no stranger things happen. Ideally this disp should fix all the issues with had with idle sprites.
